WORKSPACE.dev.bazel

```diff
@@ -22,3 +22,11 @@ load("//kotlin:repositories.bzl", "kotlin_repositories")
 kotlin_repositories()
 
 register_toolchains("@rules_kotlin//kotlin/internal:default_toolchain")
+
+load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
+load("@rules_android//android:rules.bzl", "android_sdk_repository")
+
+maybe(
```

Comment:
Mirroring what's being done in `rules_android` https://github.com/bazelbuild/rules_android/blob/main/WORKSPACE.bzlmod#L8